### PR TITLE
Complete record definition and undynamizers for get_records

### PR DESCRIPTION
**Pull request by nalundgaard**
In reference to gleber/erlcloud#189, I applied my get_records changes here. I've left the simple output in a pretty iffy state for now, but I have a proposal to change it that I will PR separately.

### CODE REVIEW

**Comment by smilerlee** [comment]
@nalundgaard Thanks very much, I'll continue based on your work :)

This branch is for preview purpose only, the final PR branch to upstream will be squashed into two or more commits as needed. One for ddb2, one for ddb_streams, others I don't know yet.

File: include/erlcloud_ddb_streams.hrl [hunk 1]
```diff
@@ -36,8 +36,24 @@
 -record(ddb_streams_describe_stream,
         {stream_description :: #ddb_streams_stream_description{}
         }).
+-record(ddb_streams_record_dynamodb,
```
Comment (smilerlee) [inline include/erlcloud_ddb_streams.hrl:39]: This record name should be `ddb_streams_stream_record` according to the [document](http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_StreamRecord.html), I've changed it.

Comment (nalundgaard) [inline include/erlcloud_ddb_streams.hrl:39]: I think that name actually corresponds to the `#ddb_streams_record{}` below, which ought perhaps to be called  `#ddb_streams_stream_record{}`. This record ought in turn to be called  `#ddb_streams_stream_record_dynamodb{}` or something. I left out the singular `stream` prefix in my pattern.

Comment (nalundgaard) [inline include/erlcloud_ddb_streams.hrl:39]: Oh, never mind, I misread the documentation for [Record](http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_Record.html) and confused it for [StreamRecord](http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_StreamRecord.html). Your change is correct.

Comment (smilerlee) [inline include/erlcloud_ddb_streams.hrl:39]: The names `ddb_streams_stream`, `ddb_streams_stream_description`, `ddb_streams_stream_record` and `ddb_streams_record` are a little complex to understand, indeed! I'll think it over.

**Pull request merged**